docs/00_what_is_cvmfs.md

```diff
@@ -1,13 +1,147 @@
 # What is CernVM-FS?
 
-(in the context of HPC)
+## In a nutshell
 
-## CernVM-FS in a nutshell
+[CernVM-FS](https://cernvm.cern.ch/fs/), the CernVM File System (also known as *CVMFS*),
+is a *file distribution service* that is particularly well suited to distribute *software installations*
+across a large number of systems world-wide in an efficient way.
 
-"streaming scientific software"
+It is implemented as a *POSIX read-only file system in user space* with [repositories](#repository)
+of files that are served via outgoing HTTP connections only, thus avoiding problems with firewalls.
+Internally, CernVM-FS uses [content-adressable storage (CAS)](https://en.wikipedia.org/wiki/Content-addressable_storage)
+and [Merkle trees](https://en.wikipedia.org/wiki/Merkle_tree) (like [Git](https://git-scm.com/) also does)
+to store file data and metadata.
+Files in a CernVM-FS repository are *automatically downloaded* to a client system as
+they are accessed, from web servers that support the CernVM-FS repository being used.
 
-incl. project history
+From an end user perspective, files in a CernVM-FS repository are available via subdirectory in `/cvmfs`,
+with a user experience similar to that of an *on-demand streaming service* for music or video,
+but then for software installations (or other types of data).
 
-## Features
+CernVM-FS was developed specifically for [distributing software](#distributing-software-as-main-use-case),
+and provides several interesting features, including
+[de-duplication of files](#de-duplication-of-files),
+[compression of data](#compression-of-data),
+[multi-level caching](#multi-level-caching),
+and [verification of data integrity](#verification-of-data-integrity),
+and has been proven to scale to hundreds of millions of files and tens of thousands of clients.
+
+
+## Use cases and features
+
+### Distributing software as main use case
+
+CernVM-FS was developed at [CERN](https://home.cern/) to let High Energy Physics (HEP) collaborations
+like the [Large Hadron Collider (LHC) project](https://home.cern/science/accelerators/large-hadron-collider)
+deploy software on the worldwide-distributed computing infrastructure used to run data processing applications,
+but can also be used to distribute large *data* repositories.
```

Comment:
It may make sense to raise the data use case not too early. It may give the audience a wrong idea. Once needs to quite carefully look at the specific use case and perhaps build a dedicated infrastructure to make cvmfs work well with data repositories.

```suggestion
deploy software on the worldwide-distributed computing infrastructure used to run data processing applications.
In certain cases, the software has also been used to distribute large *data* repositories.
```